base/client.js

```diff
@@ -227,8 +227,9 @@ const normaliseError = (error, opts, logger) => {
       err = new Error(String(error))
       break
     case 'function':
-      logger.warn(`Usage error. notify() called with a function as "error" parameter`)
-      err = new Error('Bugsnag usage error. notify() called with a function as "error" parameter')
+      const msg = 'notify() called with a function'
```

Comment:
There are a few variations on "called with" (a, an, etc) here. If space is a real issue, then using something in the following format _might_ help reduce the gzipped size, and also state the expected parameter type: "notify() expected error/opts parameters, received: $badThing"